singlevenue/src/main/java/com/mappedin/examples/singlevenue/MainActivity.java

```diff
@@ -102,6 +106,8 @@
     private boolean walking = false;
     private boolean autoRotation = false;
     private float initialDegree = 0;
+    private String[] globalMapNames = null;
```

Comment:
I dont think you should need to keep track of the map names, just get the name of the current map wheneve you switch.

Also I notice the name is still not centered between the up and down buttons